jackson-simple/src/test/java/com/baeldung/jackson/annotation/dtos/ItemTest.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.jackson.annotation.dtos;
+
+import com.baeldung.jackson.annotation.dtos.withEnum.DistanceEnumWithValue;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ItemTest {
+
+    @Test
+    public void getItem() throws JsonProcessingException {
```

Comment:
No need to add any new test class, we can include the example in the previous class only `JacksonAnnotationUnitTest.java`, also what are we trying to do here?